.gitignore

```diff
@@ -14,6 +14,7 @@
 *.nuget.props
 *.nuget.targets
 NuGet.exe
+nuget.exe
```

Comment:
Because I use bash, and git for linux treats NuGet.exe and nuget.exe as different files.